modules/nf-core/spotiflow/main.nf

```diff
@@ -0,0 +1,45 @@
+process SPOTIFLOW {
+    tag "$meta.id"
+    label 'process_medium'
+    container "docker.io/miguelib/spotiflow:0.0.1"
```

Comment:
In my opinion the container version should match the tool version - 0.0.3
Of course, best case scenario would be to have the tool on bioconda

Reply:
Unfortunately, there's not official docker file *yet* from the spotiflow developers and re-tagging is not an option.